pallets/liquidity-pools-gateway/src/tests.rs

```diff
@@ -1188,31 +1094,27 @@ mod message_processor_impl {
 					assert_eq!(mock_sender, expected_sender);
 					assert_eq!(mock_message, expected_message.serialize());
 
-					Err(DispatchErrorWithPostInfo {
+					Err(dbg!(DispatchErrorWithPostInfo {
```

Comment:
Was this a one-time change that you used when debugging or should we sprinkle it around some more? :D